docs/docs-components/SandpackExample.js

```diff
@@ -57,9 +59,9 @@ function SandpackContainer({
             <SandpackCodeEditor
               wrapContent
               style={{
-                height,
+                height: height > MIN_HEIGHT ? height : MIN_HEIGHT,
```

Comment:
I believe these are just pass-through CSS properties, so this can probably use `minHeight` and not need this logic

Reply:
height can be specified for taller examples. I set some sidenavigation examples to 500. If parallel, we want the same height for the sandpack